### DISCUSSION TITLE
Please tell me how to get the groupbox back

### DISCUSSION THREAD

**Discussion by GLC-Kazetani** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3054] [Q&A] [answered]
Hi, Everyone.
Material Design is great, but I'd like to get the groupboxes back to Windows.
How do I get it back?

[MaterialDesignGroupBox](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/blob/master/MaterialDesignThemes.Wpf/Themes/MaterialDesignTheme.GroupBox.xaml)

```xaml
            <Style x:Key="MaterialDesignGroupBox" TargetType="{x:Type GroupBox}">
                
            </Style>

```

**Comment by nicolaihenriksen** [comment, marked as answer]
@GLC-Kazetani If I understand you correctly, you are using Material Design for most parts, but want to opt-out of the `GroupBox` style?

That can be accomplished by setting the style to "null" directly on the `GroupBox` in this manner:

```xaml
<GroupBox Style="{x:Null}">
  ...
</GroupBox>
```

If you want it application wide, you will want to "null" the style for the `GroupBox` type in your `App.Xaml` or similar.

**Reply by GLC-Kazetani** [reply]
Thank you for your reply!
I was able to run
